GitVersion.yml

```diff
@@ -0,0 +1,6 @@
+continuous-delivery-fallback-tag: ci
+branches:
+    master:
+        tag: unstable
```

Comment:
isn't this missing the `ContinuousDeployment` mode?

Reply:
Cripes! Fixed